mintapi/api.py

```diff
@@ -8,15 +8,18 @@
 from mintapi import constants
 from mintapi.signIn import _create_web_driver_at_mint_com, sign_in
 from mintapi.trends import (
-    CategoryMatchFilter,
-    DateFilter,
-    DescriptionMatchFilter,
+    DateFilter as TrendDateFilter,
```

Comment:
This is done because I'm not yet sure if the Transaction Date Filter will be the same or not.  This way we can reference DateFilter in multiple classes.

Reply:
Might be cleaner to just import trends as use `trends.DateFilter` but I'm also fine with this approach!